tfquaternion/tfquaternion.py

```diff
@@ -163,8 +163,42 @@ def rotate_vector_by_quaternion(q, v, q_ndims=None, v_ndims=None):
         v = tf.expand_dims(v, axis=0) + tf.zeros_like(q_xyz)
     q_xyz += tf.zeros_like(v)
     v += tf.zeros_like(q_xyz)
-    t = 2 * tf.cross(q_xyz, v)
-    return v + tf.expand_dims(w, axis=-1) * t + tf.cross(q_xyz, t)
+    t = 2 * tf.linalg.cross(q_xyz, v)
+    return v + tf.expand_dims(w, axis=-1) * t + tf.linalg.cross(q_xyz, t)
+
+
+def quaternion_between_3d(u, v, epsilion=1e-6, dtype=tf.float32):
+    """
+    This function will help define a rotation when you know how you wish to map one specific vector
+    in a collection onto another, and apply that transformation to the whole collection.
```

Comment:
This function doesn't do anything specific to a collection, right? Could you rephrase this in terms of what this function does, and not what the potential use case might be?